### DISCUSSION TITLE
Any command to generate/preview doc before publish?

### DISCUSSION THREAD

**Discussion by Edditoria** [jsr-io/jsr#582] [Q&A]
I tried `deno doc --html` with exports properties in a deno.json file to generate static site.
However, it looks different with JSR and missing some features for ESM modules (not TS), e.g. module description, "missing-return-type" error.

Is there any command to lint and generate doc like `deno doc` in JSR? Thanks.

**Comment by cdauth** [comment]
I have the same problem and have created issue jsr-io/jsr#609 for it.
